### PR TITLE
HAAR-7: disable GraphQL endpoint; add GraphQL dev URL to pre prod and prod profile

### PR DESCRIPTION
**Pull request by neilmendum**
https://dsdmoj.atlassian.net/browse/HAAR-7

### CODE REVIEW

File: helm_deploy/values-preprod.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ generic-service:
     INGRESS_URL: "https://hmpps-audit-poc-ui-preprod.hmpps.service.justice.gov.uk"
     HMPPS_AUTH_URL: "https://sign-in-preprod.hmpps.service.justice.gov.uk/auth"
     TOKEN_VERIFICATION_API_URL: "https://token-verification-api-preprod.prison.service.justice.gov.uk"
+    GRAPHQL_API_URL: "https://gql-api-dev.hmpps.service.justice.gov.uk"
```
Comment (petergphillips) [inline helm_deploy/values-preprod.yaml:15]: wouldn't this be better set to an empty string rather that the dev value?

**Review by petergphillips — APPROVED** [review]
LGTM

File: helm_deploy/values-preprod.yaml (hunk 1 above)
Comment (neilmendum) [reply helm_deploy/values-preprod.yaml]: I thought about that but thought might fall over when trying to initalise `graphQLEndpoint`, but let's give it a go

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**